exercises/high-scores/high_scores_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require_relative 'high_scores'
+
+# Common test data version: 1.0.0 2f612cc
+class HighScoresTest < Minitest::Test
+  def test_returns_the_highest_score
+    # skip
+    assert_equal 10, HighScores.new([4, 10, 7]).highest
+  end
+
+  def test_returns_the_last_score
+    skip
+    assert_equal 3, HighScores.new([10, 0, 9, 3]).latest
+  end
+
+  def test_returns_the_top_scores_1
+    skip
+    assert_equal [5, 3, 1], HighScores.new([5, 3, 1]).top_3
```

Comment:
Not sure what the specification for this exercise is, but a method called `top_3` bugs me.
I'd expect to be able to call `top(3)`
Maybe `podium` could be a good alias?
#unsolicited_opinion :)